For some reason, this makes doxygen-1.8.6 eat all memory and segfault.

--- doc/Doxyfile.extra.in~	2015-03-07 17:02:33.000000000 +0100
+++ doc/Doxyfile.extra.in	2015-04-29 11:32:23.148101909 +0200
@@ -1,7 +1,7 @@
 INPUT = @CMAKE_SOURCE_DIR@/include \
       @CMAKE_SOURCE_DIR@/doc
 
-TAGFILES          = @HPP_MODEL_DOXYGENDOCDIR@/hpp-model.doxytag=@HPP_MODEL_DOXYGENDOCDIR@ \
+#TAGFILES          = @HPP_MODEL_DOXYGENDOCDIR@/hpp-model.doxytag=@HPP_MODEL_DOXYGENDOCDIR@ \
 		  @HPP_CONSTRAINTS_DOXYGENDOCDIR@/hpp-constraints.doxytag=@HPP_CONSTRAINTS_DOXYGENDOCDIR@
 
 HIDE_FRIEND_COMPOUNDS  = YES
